cpt/test/unit/generators_test.py

```diff
@@ -291,7 +291,7 @@ def test_visual_build_generator(self):
                                    shared_option_name=None,
                                    dll_with_static_runtime=False,
                                    vs10_x86_64_enabled=True,
-                                   build_types=["Debug", "Release"],
+                                   build_types=["Debug", "Release", "RelWithDebInfo", "MinSizeRel"],
```

Comment:
Why modifying the input of `build_types` in the test doesn't modify the "expected" generated builds?

Reply:
I was trying the make sure the `expected` array did not gain any new items. For example in this case, it should only contain `Debug` build type, even though `build_types` contain all possible values.

I can remove the *new* values. But I think it is valuable to test with new values listed and `expected` array not changing.